contracts/src/security/pausing/PauseManager.sol

```diff
@@ -132,7 +150,16 @@ abstract contract PauseManager is IPauseManager, AccessControlUpgradeable {
     if (isPaused(_pauseType)) {
       revert IsPaused(_pauseType);
     }
-
+    
+    if (hasRole(SECURITY_COUNCIL_ROLE, _msgSender())) {
+      pauseExpiry = type(uint256).max;
```

Comment:
```suggestion
      pauseExpiry = type(uint256).max - COOLDOWN_DURATION;
```

Reply:
as discussed with @kyzooghost this addresses the overflow concern on line 157

Reply:
Couldn't commit this suggest because I accepted all the other suggestions first which shifted the lines

Have made this change and adapted the test cases